src/features/wallet/send/SendView.tsx

```diff
@@ -151,53 +148,43 @@ const SendView = ({ scanResult, sendType, hotspot, isSeller }: Props) => {
 
   // process scan results
   useEffect(() => {
-    if (scanResult) {
-      if (scanResult.type !== 'transfer') {
-        setIsLocked(!!scanResult?.amount)
-        if (scanResult?.amount) {
-          const floatAmount = parseFloat(scanResult.amount)
-          const hntBalanceAmount = Balance.fromFloat(
-            floatAmount,
+    if (!scanResult) return
+    setType(scanResult.type)
+    const scannedSendTransfers: Array<SendTransfer> = scanResult.payees.map(
+      ({ address, amount: scanAmount, memo = '' }, i) => {
+        let amount = ''
+        let balanceAmount = new Balance(0, CurrencyType.networkToken)
+        if (scanAmount) {
+          balanceAmount = Balance.fromFloat(
+            parseFloat(scanAmount),
             CurrencyType.networkToken,
           )
-          const hntAmountString = hntBalanceAmount.toString(8, {
-            decimalSeparator,
-            groupSeparator,
-            showTicker: false,
-          })
-          handleAmountChange(hntAmountString)
+          amount = formatAmount(String(scanAmount))
         }
-        if (scanResult?.memo) setMemo(scanResult?.memo)
-      }
-      setType(scanResult.type)
-      setAddress(scanResult.address)
-    }
+        return {
+          id: `transfer${i}`,
+          address,
+          addressAlias: '',
+          addressLoading: false,
+          amount,
+          balanceAmount,
+          dcAmount: '',
+          memo,
+          fee: new Balance(0, CurrencyType.networkToken),
+        }
+      },
+    )
+    setSendTransfers(scannedSendTransfers)
+    const hasPresetAmount = some(scannedSendTransfers, ({ amount }) => !!amount)
+    if (hasPresetAmount) setIsLocked(true)
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [scanResult])
 
-  // compute equivalent dc amount for burn txns
-  useAsync(async () => {
-    if (type === 'dc_burn') {
-      const balanceDc = await networkTokensToDataCredits(balanceAmount)
-      if (!balanceDc) return
-
-      setDcAmount(
-        balanceDc.toString(0, {
-          decimalSeparator,
-          groupSeparator,
-          showTicker: false,
-        }),
-      )
-    }
-  }, [type, balanceAmount])
-
   // validate transaction
   useEffect(() => {
-    const isValidAddress = Address.isValid(address)
-
     if (type === 'transfer') {
+      const { fee } = sendTransfers[0]
       if (isSeller) {
-        setIsValid(isValidAddress && (hasValidActivity || false))
```

Comment:
This breaks the send button for the Hotspot Transfer form. Why did you remove it? To get to this flow you need to go to a hotspot, click settings, and then initiate a transfer of that hotspot.

Reply:
Don't think I purposefully removed this, was more so a casualty of multiple iterations 🙂  added back https://github.com/helium/hotspot-app/pull/479/commits/a97ebb18637f53dda8d0c224d3c20c6293866a70